README.md

```diff
@@ -28,6 +33,22 @@ Right now `creo2urdf` plugin needs that the dependencies are compiled and linked
 For who uses CMake integrate in Visual Studio `-DCMAKE_TOOLCHAIN_FILE` option has not to be passed to `CMake command arguments` but in the `vcpkg.cmake` file path has to be set in `CMake toolchain file`
 
 
+## Usage
+
+Create a `protk.dat` file in your CREO working directory as follows:
+
+```
+name creo2urdf
+Startup dll
+Allow_stop True
+Delay_start False
+exec_file /path/to/creo2urdf.dll
+text_dir /path/to/creo2urdf/src/creo2urdf/text
+END
+```
+
+Or use one that is automatically generated by cmake in `${PROJECT_BINARY_DIR}` (e.g. `C:\Users\ngenesio\icub-tech-iit\creo2urdf\build\x64-Release`).
```

Comment:
Probably it make sense to put this sentence first, as this is the easiest option. If I read this readme, I would first manually create the file, and then discover I could avoid to manually create it, that is a bit frustrating as a user.